docs/en/sql-reference/functions/time-series-functions.md

```diff
@@ -0,0 +1,61 @@
+---
+slug: /en/sql-reference/functions/time-series-functions
+sidebar_position: 172
+sidebar_label: Time Series
+---
+
+#Time Series Functions
+
+Below functions are used for time series analysis.
+
+## seriesDecomposeSTL
+
+Decompose time series data based on STL(Seasonal-Trend Decomposition Procedure Based on Loess)
```

Comment:
Please link STL, e.g. to https://www.wessa.net/download/stl.pdf if that's what you had in mind.

Reply:
Please use proper markdown link syntax:

```md
[...] based on [STL (Seasonal-Trend Decomposition Procedure Based on Loess)](https://www.wessa.net/download/stl.pdf) [...]
```